tempesta_fw/http.c

```diff
@@ -56,6 +57,36 @@ static struct {
 	unsigned int sz;
 } tfw_wl_marks;
 
+/* Proxy buffering size. */
+static size_t tfw_cfg_proxy_buff_sz;
+
+/* Message processing states. */
+enum {
+	/* A new message is received. */
+	Http_Msg_0 = 0,
+	/* Buffer the message before processing. */
+	Http_Msg_Buffer,
+	/* Process headers. */
+	Http_Msg_Headers,
+	/* Message is fully received. */
+	Http_Msg_End,
+	/* Forward message. */
+	Http_Msg_Fwd,
+	/* Stream message. */
+	Http_Msg_Stream,
+	/* Send response and drop connection if applicable. */
+	Http_Msg_Conn_Drop,
+};
```

Comment:
Why the state machine is separate from our current HTTP state machine with states in http.h, HTTP Generic FSM states?